docs/app/pages/charts/sections/bar-charts/bar-chart/bar-chart.component.html

```diff
@@ -0,0 +1,124 @@
+            <td><b>object[]</b></td>
+            <td>Contains the data and properties for each data set to be displayed in the chart.</td>
+        </tr>
+
+        <tr>
+            <td><b>@Input</b> chartType</td>
+            <td><b>string</b></td>
+            <td>Indicates the type of the chart. Possible options are: <code>line</code>, <code>bar</code>, <code>radar</code>, <code>pie</code>, <code>polarArea</code>, <code>doughnut</code></td>
+        </tr>
+
+        <tr>
+            <td><b>@Input</b> options</td>
+            <td><b>ChartOptions</b></td>
+            <td>Configures the appearance and behavior of all aspects of the chart.</td>
+        </tr>
+
+        <tr>
+            <td><b>@Input</b> colors</td>
+            <td><b>object[]</b></td>
+            <td>Configure the color values to be used within the chart.</td>
+        </tr>
+
+        <tr>
+            <td><b>@Input</b> legend</td>
+            <td><b>boolean</b></td>
+            <td>Determines whether or not the legend should be displayed.</td>
+        </tr>
+
+        <tr>
+            <td><b>@Output</b> chartClick</td>
+            <td></td>
+            <td>Fires when click on a chart has occurred, returns information regarding active points and labels</td>
+        </tr>
+
+        <tr>
+            <td><b>@Output</b> chartHover</td>
+            <td></td>
+            <td>Fires when mousemove (hover) on a chart has occurred, returns information regarding active points and labels.</td>
+        </tr>
+
+    </tbody>
+</table>
+
+<p>Complete documentation on the directive and chart options can be found at the documentation links below.</p>
+
+<p>The following code can be used to create the example above: </p>
+
+<tabset>
+
+    <tab heading="HTML">
+        <uxd-snippet language="html" [code]="htmlCode"></uxd-snippet>
+    </tab>
+    
+    <tab heading="TypeScript">
+        <uxd-snippet language="javascript" [code]="tsCode"></uxd-snippet>
+    </tab>
+    
+    <tab heading="CSS">
+        <uxd-snippet language="css" [code]="cssCode"></uxd-snippet>
+    </tab>
+
+</tabset>
+
+<br>
+
+<blockquote>
+
+    <p class="m-b-nil">
+        <b>Dependencies: </b>
+        <a class="dependency-link" href="https://www.npmjs.com/package/ng2-charts">ng2-charts</a>, 
+        <a class="dependency-link" href="https://www.npmjs.com/package/chart.js">chart.js</a>
+    </p>
+
+    <p class="m-b-nil">
+        <b>Type Definitions: </b>
+        <a class="dependency-link" href="https://www.npmjs.com/package/@types/chart.js">@types/chart.js</a>
+    </p>
+
+    <p class="m-b-nil">
+        <b>Modules: </b>
+        <span class="dependency-module">ChartsModule</span>
+    </p>
+
+
+    <p class="m-b-nil">
+        <b>Documentation: </b>
+        <a class="dependency-link" href="http://valor-software.com/ng2-charts/">Angular Directive Documentation</a>, 
+        <a class="dependency-link" href="http://www.chartjs.org/docs/">Chart Library Documentation</a>
```

Comment:
These hyperlinks are not visually distinctive, maybe use standard text and hyperlink styles here.